### DISCUSSION TITLE
Mono Theme Newsletter

### DISCUSSION THREAD

**Discussion by Sunil-D-R** [GetPublii/Publii#1432] [General]
I can't seem to find the headline for the newsletter section in the mono theme. See screenshot
<img width="301" alt="image" src="https://github.com/GetPublii/Publii/assets/142221930/d9f7cbc1-6dec-495d-8b14-56feeadd156c">

**Comment by bobmitro** [comment]
Navigate to the Sidebar tab in the theme settings.

**Comment by Sunil-D-R** [comment]
Yes I got that part as you can see in the previous screenshot. But the text part I can seem to find where to make the change.
<img width="301" alt="image" src="https://github.com/GetPublii/Publii/assets/142221930/a089548f-2f36-4d24-9b59-1d5ce824030c">

**Comment by bobmitro** [comment]
<p>All words from the theme may be found in the language file.<br />
For more useful information, where to find the file, as well as how to prevent overwriting of changes during theme update, please refer to our documentation: https://getpublii.com/docs/translate-publii-theme-to-another-language.html</p>

**Comment by Sunil-D-R** [comment]
Thanks. Maybe i misunderstood but the link you sent is about translating the theme and makes no mention of anything about the newsletter.

**Reply by candidexmedia** [reply]
Hi @Sunil-D-R ! What Bob means is that the text is set inside the theme's lang file `mono.lang.json`.

# Option 1: `lang.json`

You can find this file by going to [[Publii sites directory](https://getpublii.com/docs/publii-global-settings.html#fileslocation)] > [site folder] > input > [theme or [theme-override](https://getpublii.com/dev/theme-overrides/) folder]:

![image](https://github.com/GetPublii/Publii/assets/95003834/505ba2c1-9390-4838-ba9d-1e4543a58434)

From there, you can edit the lines above in the `mono.lang.json` file, using a text editor.

# Option 2: `sidebar.hbs`

Alternately, you can also edit the text in the `sidebar.hbs` file. 

You can find this file here:

 [[Publii sites directory](https://getpublii.com/docs/publii-global-settings.html#fileslocation)] > [site folder] > input > [theme or [theme-override](https://getpublii.com/dev/theme-overrides/) folder] > partials

To edit the title, swap out the highlighted text below with your chosen text:

![image](https://github.com/GetPublii/Publii/assets/95003834/feeb3ec1-38eb-4554-b17b-f2336cf4171c)

So, for example, it would go from this:

```handlebars
<h3 class="box__title">{{ translate 'partials.sidebar.newsletter' }}</h3>
```

to this:

```handlebars
<h3 class="box__title">My new title</h3>
```

To edit the description, swap out the following text:

![image](https://github.com/GetPublii/Publii/assets/95003834/b79d58c7-b632-48bb-91eb-d6579c57f0a9)


When making any changes to the base theme files, it's highly recommended to create a theme override folder: https://getpublii.com/dev/theme-overrides/

Hope this helps!
